functionary/ui/templates/forms/team/teamuserrole_create_or_update.html

```diff
@@ -0,0 +1,71 @@
+{% extends "partials/modal.html" %}
```

Comment:
I'm guessing this was intended to be deleted.

Reply:
Fixed